src/view/signInView/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './signInView'
```

Comment:
What for do you need to create index.js and export default every time?

Is it for const SignInView = lazy(() =>
  import('./view/signInView' /* webpackChunkName: "signin-page" */)
)???

Reply:
it's for shorter imports (good practice)